resources/views/pilots/message-history.blade.php

```diff
@@ -4,4 +4,11 @@
         <h1 class="fs-2 mb-4 font-display text-primary-emphasis">Message history</h1>
         <livewire:pilots.message-history/>
     </div>
+{{--    <script type="module">--}}
+{{--        let id = {{ auth()->id() }}--}}
+{{--        Echo.private(`clearance.${id}`)--}}
+{{--            .listen('.clx.issued', (e) => {--}}
+{{--                console.log(e)--}}
+{{--            })--}}
+{{--    </script>--}}
```

Comment:
was this still needed or do the event handlers in `livewire.pilots.message-history` take care of it

Reply:
Not needed